relayer/workers/beefyrelayer/beefy-ethereum-writer.go

```diff
@@ -0,0 +1,187 @@
+package beefyrelayer
+
+import (
+	"context"
+	"fmt"
+
+	"golang.org/x/sync/errgroup"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/sirupsen/logrus"
+
+	"github.com/snowfork/polkadot-ethereum/relayer/chain/ethereum"
+	"github.com/snowfork/polkadot-ethereum/relayer/contracts/lightclientbridge"
+	"github.com/snowfork/polkadot-ethereum/relayer/workers/beefyrelayer/store"
+)
+
+type BeefyEthereumWriter struct {
+	ethereumConfig    *ethereum.Config
+	ethereumConn      *ethereum.Connection
+	beefyDB           *store.Database
+	lightClientBridge *lightclientbridge.Contract
+	databaseMessages  chan<- store.DatabaseCmd
+	beefyMessages     <-chan store.BeefyRelayInfo
+	log               *logrus.Entry
+}
+
+func NewBeefyEthereumWriter(ethereumConfig *ethereum.Config, ethereumConn *ethereum.Connection, beefyDB *store.Database,
+	databaseMessages chan<- store.DatabaseCmd, beefyMessages <-chan store.BeefyRelayInfo,
+	log *logrus.Entry) (*BeefyEthereumWriter, error) {
```

Comment:
```suggestion
	log *logrus.Entry) *BeefyEthereumWriter {
```
Error is always nil, are we able to remove `error` from the return values?